pkg/apis/crd.nsx.vmware.com/v1alpha1/ipblocksinfo_types.go

```diff
@@ -0,0 +1,40 @@
+/* Copyright © 2024 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Should these be Broadcom copyrights now? cc @jianjuns

Reply:
Updated, thanks